libc/utils/MPFRWrapper/MPFRUtils.cpp

```diff
@@ -30,6 +31,10 @@ namespace mpfr {
 // precision compared to the floating point precision.
 template <typename T> struct ExtraPrecision;
 
+template <> struct ExtraPrecision<float16> {
+  static constexpr unsigned int VALUE = 128;
```

Comment:
I didn't know what value to use here, so I just copied the one for `float`.

Reply:
I think that is ok.